core_impl/src/test/java/io/opencensus/implcore/stats/ViewManagerImplTest.java

```diff
@@ -199,17 +201,27 @@ public void disallowGettingNonexistentViewData() {
   }
 
   @Test
-  public void testRecordDouble_cumulative() {
-    testRecordCumulative(MEASURE_DOUBLE, 10.0, 20.0, 30.0, 40.0);
+  public void testRecordDouble_distribution_cumulative() {
+    testRecordCumulative(MEASURE_DOUBLE, DISTRIBUTION, 10.0, 20.0, 30.0, 40.0);
   }
 
   @Test
-  public void testRecordLong_cumulative() {
-    testRecordCumulative(MEASURE_LONG, 1000, 2000, 3000, 4000);
+  public void testRecordLong_distribution_cumulative() {
+    testRecordCumulative(MEASURE_LONG, DISTRIBUTION, 1000, 2000, 3000, 4000);
   }
 
-  private void testRecordCumulative(Measure measure, double... values) {
-    View view = createCumulativeView(VIEW_NAME, measure, DISTRIBUTION, Arrays.asList(KEY));
+  @Test
+  public void testRecordDouble_sum_cumulative() {
+    testRecordCumulative(MEASURE_DOUBLE, SUM, 10.0, 20.0, 30.0, 40.0);
```

Comment:
I think it would help to include some non-integers in this test, to check that they aren't rounded.

Reply:
Done.